holoplot/converter.py

```diff
@@ -546,6 +455,14 @@ def __call__(self, kind, x, y):
             eltype = 'Image'
             if 'cmap' in self._style_opts:
                 style['cmap'] = self._style_opts['cmap']
+
+        if self.crs:
+            # Apply projection before rasterizing
+            import cartopy.crs as ccrs
+            from geoviews import project
+            projection = self._plot_opts.get('projection', ccrs.GOOGLE_MERCATOR)
+            obj = project(obj, projection=projection)
```

Comment:
Web Mercator is the default output projection, not Plate Carree?

Reply:
I guess that way the projection won't change if someone overlays it with a tile source, and I suppose it's arbitrary anyway....

Reply:
>Web Mercator is the default output projection, not Plate Carree?

For bokeh yes, for matplotlib no, but holoplot is focused on bokeh output for now.

>I guess that way the projection won't change if someone overlays it with a tile source

That's right, we picked GOOGLE_MERCATOR as the default for bokeh as that was the only way tile sources would work, and now we're somewhat restricted by backward compatibility.